app/com/linkedin/drelephant/tuning/TuningHelper.java

```diff
@@ -122,4 +125,15 @@ public static void updateJobExecution(JobExecution jobExecution) {
       logger.error("Exception updating job execution " + jobExecution.id, e);
     }
   }
+
+  public static Double getContainerSize(Double memory) {
+    return Math.ceil(memory / ONE_GB) * ONE_GB;
```

Comment:
Yes in your PR which is not merged yet so I couldn't use them, so have implemented for now and at the time of merge I will resolve the conflicts and this will be removed.

Reply:
ok . Make sense